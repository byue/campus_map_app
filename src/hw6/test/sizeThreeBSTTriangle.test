# Simple test of breadth-first search algorithm.
# Verifies output of BFS for size 3 triangle graph.

CreateGraph characters
AddNode characters GORILLA-MAN
AddNode characters HAWK
AddNode characters VENUS-II

AddEdge characters GORILLA-MAN HAWK COMIC1
AddEdge characters HAWK VENUS-II COMIC2
AddEdge characters VENUS-II GORILLA-MAN COMIC3

FindPath characters GORILLA-MAN HAWK

FindPath characters HAWK VENUS-II

FindPath characters GORILLA-MAN VENUS-II

FindPath characters VENUS-II GORILLA-MAN