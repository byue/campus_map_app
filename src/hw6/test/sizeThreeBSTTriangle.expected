# Simple test of breadth-first search algorithm.
# Verifies output of BFS for size 3 triangle graph.

created graph characters
added node GORILLA-MAN to characters
added node HAWK to characters
added node VENUS-II to characters

added edge COMIC1 from GORILLA-MAN to HAWK in characters
added edge COMIC2 from HAWK to VENUS-II in characters
added edge COMIC3 from VENUS-II to GORILLA-MAN in characters

path from GORILLA-MAN to HAWK:
GORILLA-MAN to HAWK via COMIC1

path from HAWK to VENUS-II:
HAWK to VENUS-II via COMIC2

path from GORILLA-MAN to VENUS-II:
GORILLA-MAN to HAWK via COMIC1
HAWK to VENUS-II via COMIC2

path from VENUS-II to GORILLA-MAN:
VENUS-II to GORILLA-MAN via COMIC3